paparazzi/src/main/java/app/cash/paparazzi/PaparazziSdk.kt

```diff
@@ -339,6 +340,12 @@ public class PaparazziSdk @JvmOverloads constructor(
       if (hasComposeRuntime) {
         forceReleaseComposeReferenceLeaks()
       }
+
+      // Reset the choreographer to its initial state for last for future test runs as it is a singleton.
+      val choreographer = Choreographer.getInstance()
+      val mLastFrameTimeNanos = choreographer::class.java.getDeclaredField("mLastFrameTimeNanos")
+      mLastFrameTimeNanos.isAccessible = true
+      mLastFrameTimeNanos.set(choreographer, 0L)
```

Comment:
@jrodbx was able to step through and see what was going on. Since Choreographer is a singleton, that last frame time carries over from previous tests, causing animation timing issues.

Thought it would be a good clean up step to reset it to 0 after snapshot. Could also do it before in the prepare step.